models/slice/matplotlib_slice_plotter.py

```diff
@@ -0,0 +1,47 @@
+from slice_plotter import SlicePlotter
+from mantid.simpleapi import mtd
+from plotting import pyplot as plt
+import numpy as np
+
+
+def get_aspect_ratio(self, workspace):
+        return 'auto'
+
+
+class MatplotlibSlicePlotter(SlicePlotter):
+    def display_slice(self, selected_workspace, x_axis, y_axis, smoothing, intensity_start, intensity_end, norm_to_one,
+                      colourmap):
+        workspace = mtd[selected_workspace]
```

Comment:
This is quite picky but can we avoid using `mtd` and using `AnalysisDataService` instead? I think libraries such as this should be verbose as possible and `mtd` reserved for short hand in user space.